go-controller/pkg/node/gateway_iptables.go

```diff
@@ -243,10 +243,10 @@ func getExternalIPTRules(svcPort kapi.ServicePort, externalIP, dstIP string) []i
 	}
 }
 
-func getLocalGatewayNATRules(ifname string, ip net.IP) []iptRule {
+func getLocalGatewayNATRules(ifname string, cidr *net.IPNet) []iptRule {
 	// Allow packets to/from the gateway interface in case defaults deny
 	var protocol iptables.Protocol
-	if utilnet.IsIPv6(ip) {
+	if utilnet.IsIPv6(cidr.IP) {
```

Comment:
@trozet this whole hunk seems uncontroversial and could just be merged almost immediately.